dockerfiles/base/scripts/base/commands/cmd_start.sh

```diff
@@ -112,7 +112,50 @@ cmd_restart() {
   debug $FUNCNAME
 
   FORCE_UPDATE=${1:-"--no-force"}
-    info "restart" "Restarting..."
-    cmd_stop
-    cmd_start ${FORCE_UPDATE}
+  info "restart" "Restarting..."
+  cmd_stop
+  cmd_start ${FORCE_UPDATE}
 }
+
+check_if_booted() {
+  CURRENT_CHE_SERVER_CONTAINER_ID=$(get_server_container_id $CHE_SERVER_CONTAINER_NAME)
+  wait_until_container_is_running 20 ${CURRENT_CHE_SERVER_CONTAINER_ID}
+  if ! container_is_running ${CURRENT_CHE_SERVER_CONTAINER_ID}; then
+    error "(${CHE_MINI_PRODUCT_NAME} start): Timeout waiting for ${CHE_MINI_PRODUCT_NAME} container to start."
+    return 2
+  fi
+
+  info "start" "Services booting..."
+
+  # CHE-3546 - if in development mode, then display the che server logs to STDOUT
+  #            automatically kill the streaming of the log output when the server is booted
+  if debug_server; then
+    docker logs -f ${CHE_SERVER_CONTAINER_NAME} &
+    LOG_PID=$!
+  else
+    info "start" "Server logs at \"docker logs -f ${CHE_SERVER_CONTAINER_NAME}\""
+  fi
+
+  wait_until_server_is_booted 60 ${CURRENT_CHE_SERVER_CONTAINER_ID}
+ 
+  if server_is_booted ${CURRENT_CHE_SERVER_CONTAINER_ID}; then
+    if debug_server; then
+      kill $LOG_PID > /dev/null 2>&1
+      info ""
+    fi
+
+    DISPLAY_URL=$(get_display_url)
+    info "start" "Booted and reachable"
+    info "start" "Ver: $(get_installed_version)"
+    info "start" "Use: ${DISPLAY_URL}"
+    info "start" "API: ${DISPLAY_URL}/swagger"
+    if debug_server; then
+      DISPLAY_DEBUG_URL=$(get_debug_display_url)
+      info "start" "Debug: ${DISPLAY_DEBUG_URL}"
+    fi
+  else
+    error "(${CHE_MINI_PRODUCT_NAME} start): Timeout waiting for server. Run \"docker logs ${CHE_SERVER_CONTAINER_NAME}\" to inspect the issue."
```

Comment:
Moved the kill operation to be outside of hte initial loop so that it's killed first, displayed second.